lib/katello/tasks/check_config.rake

```diff
@@ -0,0 +1,17 @@
+namespace :katello do
+  task :check_config => ['environment'] do
+    desc "Task that can be run before a content migration to check that the configuration valid"
+
+    puts "Checking for valid Katello configuraton."
+    if SETTINGS[:katello][:use_pulp_2_for_content_type].nil?
+      fail _("Invalid Katello configuration: 'use_pulp_2_for_content_type' is missing. ")
+    end
+
+    if !SETTINGS[:katello][:use_pulp_2_for_content_type][:docker] &&
+      !SETTINGS[:katello][:use_pulp_2_for_content_type][:file] &&
+      !SETTINGS[:katello][:use_pulp_2_for_content_type][:yum] &&
+      !SETTINGS[:katello][:use_pulp_2_for_content_type][:deb]
+      fail _("Invalid Katello configuration: no content types in 'use_pulp_2_for_content_type' section are set to 'true'. ")
```

Comment:
we might wanna adjust these error messages a bit, something more along the lines of "The system appears to already be using pulp3 with all content migrated".  From the user perspective, these settings are kinda meaningless.  

I would adjust the one above to kinda same the same thing.  That section missing isn't invalid, it just means the content has already been migrated and switched over.  I think the same message for both of these is fine.